scalafix/rules/src/main/scala/rsc/rules/semantics/Symtabs.scala

```diff
@@ -2,6 +2,34 @@
 // Licensed under the Apache License, Version 2.0 (see LICENSE.md).
 package rsc.rules.semantics
 
+import scala.meta.internal.{semanticdb => s}
+import scala.meta.internal.semanticdb.Scala.{Names => n}
+
 trait Symtabs {
   type Symtab = scala.meta.internal.symtab.SymbolTable
+
+  implicit class SymtabOps(symtab: Symtab) {
+
+    def equivalent(env: Env, name: Option[n.Name], sym: String): Boolean = {
+      name
+        .flatMap(x => symtab.info(env.lookup(x)))
```

Comment:
How about doing a lookup at the callsite so that this method doesn't know anything about env and looks like `def equivalent(sym1: String, sym2: String): Boolean`?